### PR TITLE
Feature/region

### PR DESCRIPTION
**Pull request by bengarvey**
This branch adds a lookup table for region ids and adds them to the magento data.

##### Functional tests
- run `coffee generate.coffee` and make sure regions and country_ids are added to the address data instead of state and country.

### CODE REVIEW

File: lib/region_id.json [hunk 1]
```diff
@@ -0,0 +1,68 @@
+{ "region_id": {
```
Comment (ynicolaou) [inline lib/region_id.json]: Why not use `chance.state()` instead?

File: generate.coffee [hunk 2]
```diff
@@ -86,8 +87,8 @@ generateAddresses = (total) ->
     address =
       entity_id: index
       city: chance.city()
-      state: chance.state()
-      country: chance.country({full:true})
+      region: regions.region_id[chance.state()]
```
Comment (ynicolaou) [inline generate.coffee]: I don't think you need a region id here. This field is a varchar and from querying a magento client's DB i get this kind of data:

```
entity_id	country_id	region
1	US	Alabama
2	US	Alabama
23	US	New York
24	US	New York
25	US	New York
26	US	New York
27	US	New York
28	US	New York
29	US	New York
30	US	New York
31	US	North Carolina
32	US	North Carolina
33	US	Oregon
34	US	Oregon
35	US	Oregon
36	US	Oregon
```

Comment (bengarvey) [inline generate.coffee]: I'm going to switch the generator to be US only and region to be a random state for now.

**Comment by bengarvey** [comment]
@ynicolaou back to you.

**Pull request merged**